File: Beecrowd/Beginner/2203 - ~~/2313 - Which Triangle/Python/2313_WhichTriangle.py
a, b, c = map(int, input().split())

if a < b + c and b < a + c and c < a + b:
    print("Valido-", end="")

    if a == b == c:
        print("Equilatero")
    elif a == b or b == c or a == c:
        print("Isoceles")
    else:
        print("Escaleno")

    print("Retangulo: ", end="")

    if a * a == b * b + c * c or b * b == a * a + c * c or c * c == a * a + b * b:
        print("S")
    else:
        print("N")
else:
    print("Invalido")

File: Beecrowd/Beginner/2203 - ~~/2313 - Which Triangle/C/2313_WhichTriangle.c
#include <stdio.h>

int main() {
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);

    if (a < b + c && b < a + c && c < a + b) {
        printf("Valido-");

        if (a == b && b == c) {
            printf("Equilatero\n");
        } else if (a == b || b == c || a == c) {
            printf("Isoceles\n");
        } else printf("Escaleno\n");

        printf("Retangulo: ");

        if (a * a == b * b + c * c ||
            b * b == a * a + c * c ||
            c * c == a * a + b * b) {
            printf("S\n");
        } else printf("N\n");
    } else printf("Invalido\n");

    return 0;
}
